### PR TITLE
feat: Add various legal and security resources

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Comment by dcramer** [comment]
Feels like on content, but I'm not sure the solution that ensures its discoverable still..

image

**Comment by dcramer** [comment]
Not in love with this as it feels.. empty. If folks have thoughts on how we can resolve that, or what other resources we can add, let me know.

**Comment by dcramer** [comment]
Aside Terms of Service isn't clear on what it means. Maybe would fit better as "Legal Terms" I'm not sure though.

File: nginx.conf [hunk 1]
```diff
@@ -34,6 +34,8 @@ server {
   rewrite ^/server/(.*)$ https://github.com/getsentry/onpremise/ redirect;
   # /self-hosting redirects
 
+  rewrite ^/ip-ranges/?$ /meta/ip-ranges/$is_args$args redirect;
```
Comment (dcramer) [inline nginx.conf:37]: of note I threw this under meta as 'security-and-legal' is awfully chunky

**Comment by dcramer** [comment]
Aside it would have been nice to add more links to the sidebar, but its auto generated so quite tricky to do non-page links. e.g. adding the privacy shield link maybe, or adding the security overview link.

File: nginx.conf (hunk 1 above)
Comment (PeloWriter) [reply nginx.conf:37]: Agreed. 

Of note: GitHub provides the DPA as part of the organization settings information - we might consider that for future. They also create a step-by-step article on how to accept it: https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/entering-a-data-protection-agreement-with-github-for-gdpr-compliance

They also require that the the Terms of Service have been accepted first; do we?

File: src/collections/_documentation/meta/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: 'Security & Legal'
+sidebar_order: 5
+---
+
+Sentry provides self-serve access to various documentation and tools to help you with your security and compliance processes.
```
Comment (PeloWriter) [inline src/collections/_documentation/meta/index.md:6]: Suggest adding: "Please contact us at sales@sentry.io if you have questions or need clarification. 

When you’re logged in to your Sentry account, you can review and accept these agreements as an authorized individual on behalf of your organization."

File: src/collections/_documentation/meta/terms.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: 'Terms of Service'
+sidebar_order: 1
+---
+
+Sentry provides a way to self-serve most legal and compliance documentation for your organization. You'll find these by navigating to _Your Organization's Settings_ > Legal & Compliance.
```
Comment (PeloWriter) [inline src/collections/_documentation/meta/terms.md:6]: Suggest adding: "When you’re logged in to your Sentry account, you can review and accept these agreements as an authorized individual on behalf of your organization."

File: src/collections/_documentation/meta/terms.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+![Legal & Compliance Settings]({% asset legal-and-compliance.png @path %})
+
+Available legal documents include:
+
+- [Terms of Service](https://sentry.io/terms/)
+- [Privacy Policy](https://sentry.io/privacy/)
+- [Data Processing Amendment (DPA)](https://sentry.io/legal/dpa/1.0.0/)
+- [California Consumer Privacy Act Addendum (CCPA)](https://sentry.io/legal/ccpa/1.0.0/)
+
+You will need to accept agreements as an authorized individual on behalf of your organization.
```
Comment (PeloWriter) [inline src/collections/_documentation/meta/terms.md:17]: remove if you are adding the sentence to line 6.

File: src/collections/_documentation/workflow/integrations/global-integrations.md [hunk 5]
```diff
@@ -230,7 +230,7 @@ openssl x509 -pubkey -noout -in bitbucket_publickey.cer  > bitbucket_publickey.p
 
 ##### III. Connect your Bitbucket Server application with Sentry
 {% capture __alert_content -%}
-Make sure you have whitelisted [Sentry's IP ranges.](https://docs.sentry.io/ip-ranges/)
+Make sure you have whitelisted [Sentry's IP ranges]({%- link _documentation/meta/ip-ranges.md -%}).
```
Comment (PeloWriter) [inline src/collections/_documentation/workflow/integrations/global-integrations.md:233]: Nit: change "Make sure you have..." to "Confirm [Sentry's IP ranges] ({%- link _documentation/meta/ip-ranges.md -%}) are whitelisted" 

Or, more directive "Whitelist [Sentry's IP ranges] ({%- link _documentation/meta/ip-ranges.md -%})"

File: src/collections/_documentation/workflow/integrations/global-integrations.md [hunk 6]
```diff
@@ -458,7 +458,7 @@ Sentry owner or manager permissions, and GitHub owner permissions are required t
 
 ###### Add new GitHub App
 
-1. Make sure you've whitelisted Sentry’s outbound request [IPs addresses](https://docs.sentry.io/ip-ranges/) for your GitHub Enterprise instance.
+1. Make sure you've whitelisted Sentry’s outbound request [IPs ranges]({%- link _documentation/meta/ip-ranges.md -%}) for your GitHub Enterprise instance.
```
Comment (PeloWriter) [inline src/collections/_documentation/workflow/integrations/global-integrations.md:461]: see previous nit for "Make sure"

File: src/collections/_documentation/workflow/integrations/global-integrations.md [hunk 7]
```diff
@@ -815,7 +815,7 @@ openssl x509 -pubkey -noout -in jira_publickey.cer  > jira_publickey.pem
 
 ##### III. Connect your Jira Server application with Sentry
 {% capture __alert_content -%}
-Make sure you have whitelisted [Sentry's IP ranges.](https://docs.sentry.io/ip-ranges/)
+Make sure you have whitelisted [Sentry's IP ranges]({%- link _documentation/meta/ip-ranges.md -%}).
```
Comment (PeloWriter) [inline src/collections/_documentation/workflow/integrations/global-integrations.md:818]: See previous nit for "Make sure"

File: nginx.conf (hunk 1 above)
Comment (dcramer) [reply nginx.conf:37]: this is similar to what we do except that we are pure digital

**Pull request merged**